src/features/student/views/StudentTraceView/components/UpdateTraceModal/UpdateTraceModal.vue

```diff
@@ -23,25 +23,34 @@ function goToNextStep () {
   currentStep.value++
 }
 
+async function handleConfirm () {
+  if (currentStep.value === 0) {
+    goToNextStep()
+  }
+  else {
+    await tracesStore.submitUpdateTraceForm()
+  }
+}
+
 const displayedStep = computed(() => currentStep.value === 0 ? TermsStep : UpdateStep)
-const onConfirm = computed(() => currentStep.value === 0 ? goToNextStep : onClose)
 const confirmLabel = computed(() => currentStep.value === 0
   ? t('student.views.studentTraceView.updateTraceModal.buttons.validate')
   : t('student.views.studentTraceView.updateTraceModal.buttons.save'))
+
 const confirmIcon = computed(() => currentStep.value === 0
   ? MDI_ICONS.CHECK_CIRCLE_OUTLINE
   : MDI_ICONS.CONTENT_SAVE_OUTLINE)
 </script>
 
 <template>
   <AvModal
-    :opened="show"
-    :close-button-label="t('student.views.studentTraceView.updateTraceModal.buttons.close')"
+    :opened="showUpdateTraceModal"
+    :close-button-label="t('student.views.studentTraceView.updateTraceModal.buttons.close').toUpperCase()"
```

Comment:
```suggestion
    :close-button-label="t('student.views.studentTraceView.updateTraceModal.buttons.close')"
```
On a uniformisé les boutons, plus d'uppercase dedans